products/securityscanner/terraform.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  ScanConfig: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["{{name}}"]
+    properties:
+      authentication: !ruby/object:Overrides::Terraform::PropertyOverride
+        properties:
+          googleAccount: !ruby/object:Overrides::Terraform::PropertyOverride
+            properties:
+              password: !ruby/object:Overrides::Terraform::PropertyOverride 
+                sensitive: true
+          customAccount: !ruby/object:Overrides::Terraform::PropertyOverride
+            properties:
+              password: !ruby/object:Overrides::Terraform::PropertyOverride 
+                sensitive: true
+      schedule: !ruby/object:Overrides::Terraform::PropertyOverride
+        properties:
+          scheduleTime: !ruby/object:Overrides::Terraform::PropertyOverride
+            default_from_api: true
+    examples:
```

Comment:
super nit, mostly just a note for the future and you don't need to fix here: `examples` goes before `properties` generally